### PR TITLE
Extend Deeply Read switch until 5th Feb

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

Extends the Deeply Read switch until the 5th Feb 21.

And some linting by the look of it.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/discussion/comment-box.js [hunk 1]
```diff
@@ -87,9 +87,7 @@ class CommentBox extends Component {
                 'Sorry, your comment was not published as you are no longer signed in. Please sign in and try again.',
             'READ-ONLY-MODE':
                 'Sorry your comment can not currently be published as commenting is undergoing maintenance but will be back shortly. Please try again in a moment.',
-            USERNAME_MISSING: `You must <a href="${config.get(
-                'page.mmaUrl'
-            )}/public-settings">set a username</a> before commenting. (<a href="/help/2020/feb/10/why-am-i-unable-to-post-a-comment">Learn more</a>).`,
+            USERNAME_MISSING: `You must <a href="${config.get('page.mmaUrl')}/public-settings">set a username</a> before commenting. (<a href="/help/2020/feb/10/why-am-i-unable-to-post-a-comment">Learn more</a>).`,
```
Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/discussion/comment-box.js:90]: Looks like this was accidently commited?

File: common/app/conf/switches/ABTestSwitches.scala [hunk 2]
```diff
@@ -41,7 +41,7 @@ trait ABTestSwitches {
     "Tests an onward hypothesis by replacing the second tab in the Most Popular container with deeply read items.",
     owners = Seq(Owner.withGithub("nitro-marky")),
     safeState = Off,
-    sellByDate = new LocalDate(2021, 1, 29),
+    sellByDate = new LocalDate(2021, 2, 5),
```
Comment (gtrufitt) [inline common/app/conf/switches/ABTestSwitches.scala:44]: 👍

**Review by gtrufitt — APPROVED** [review]
Had a rouge change sneak in but otherwise 👍

File: static/src/javascripts/projects/common/modules/discussion/comment-box.js (hunk 1 above)
Comment (nitro-marky) [reply static/src/javascripts/projects/common/modules/discussion/comment-box.js:90]: I can only guess it was linted when I was pushing(?)

Comment (gtrufitt) [reply static/src/javascripts/projects/common/modules/discussion/comment-box.js:90]: I think you made this change the other day, it probably came over from your main branch

**Pull request merged**